src/arrayMethodSort.js

```diff
@@ -3,10 +3,39 @@
 /**
  * Implement method Sort
  */
-function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
-  };
+function applyCustomSort(a, b) {
+  const strA = String(a);
+  const strB = String(b);
+
+  if (strA > strB) {
```

Comment:
```suggestion
  if (a.toString() > b.toString) {
```

You can simplify your code here